server/aws-lsp-codewhisperer/src/language-server/agenticChat/context/contextCommandsProvider.ts

```diff
@@ -125,11 +129,25 @@ export class ContextCommandsProvider implements Disposable {
             description: 'Add a saved prompt to context',
             icon: 'magic',
         }
-        const allCommands = [
+        const commands = [
+            ...(localProjectContextEnabled
```

Comment:
I'll probably remove this check with the following PRs, prompting customer to enable the indexing lib. For now, I think it's safer to keep the check here.